### DISCUSSION TITLE
pihieating_demand not exposed on Eurotronic Spirit zigbee

### DISCUSSION THREAD

**Discussion by peyoz** [Koenkk/zigbee2mqtt#5557] [Devices]
Hi Guys,
I'm very new to the environment so maybe I'm missing some bits, but why is the pi_heating_demand attribute not exposed on the eurotronic spirit zigbee?
I easily added the attribute as a sensor on homeassistant, but I'm wondering why is not exposed directly from zigbee2mqtt.

![image](https://user-images.githubusercontent.com/61762607/103567568-56197e80-4ec4-11eb-9e4d-0f36f1bce4d6.png)

State looks fine to me
```
{
    "battery": 100,
    "current_heating_setpoint": 5,
    "eurotronic_error_status": 0,
    "eurotronic_host_flags": {
        "boost": false,
        "child_protection": false,
        "mirror_display": false,
        "window_open": false
    },
    "eurotronic_system_mode": 1,
    "linkquality": 185,
    "local_temperature": 22.5,
    "occupied_heating_setpoint": 5,
    "pi_heating_demand": 0,
    "system_mode": "auto",
    "unoccupied_heating_setpoint": 16
}
```
It's pretty useful to understand the "usage" of the valve

**Comment by Koenkk** [comment]
Added! 

Changes will be available in the latest dev branch in a few hours (https://www.zigbee2mqtt.io/how_tos/how-to-switch-to-dev-branch.html)

**Reply by peyoz** [reply]
Awesome, Many Thanks!
